js/ooyala_ssai.js

```diff
@@ -342,12 +347,15 @@ OO.Ads.manager(function(_, $)
      */
     this.resumeAd = function(ad)
     {
-      if (ad && ad.ad && ad.ad.data && this.adIdDictionary[ad.ad.data.id] && currentId3Object) {
```

Comment:
Just double checking, do we no longer care about the existence of currentId3Object in this function?

Reply:
@aeng7 this was actually a bug from before I think... there is no `currentId3Object` variable in scope here. The goal of this is to get the duration of the ad to restart the ad timer with that duration. The `ad` object itself should contain this at this point so we do not actually need it even